src/SmartComponents/Drift/DriftDropDown.js

```diff
@@ -4,22 +4,22 @@ import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { translateDriftDropdownItems } from './utils';
 
+import { CaretDownIcon } from '@patternfly/react-icons';
 import {
     Dropdown,
-    DropdownToggle,
     DropdownItem
-} from '@patternfly/react-core';
-import CaretDownIcon from '@patternfly/react-icons/dist/esm/icons/caret-down-icon';
+} from '@patternfly/react-core/deprecated';
+import { DropdownToggle } from '@patternfly/react-core/dist/esm/deprecated/components/Dropdown';
```

Comment:
Relative imports are neede here as well :)

Reply:
I thought they fixed the bundling issue with the pf5 upgrades?

Reply:
the new ts-transformer does this for us. For more info: https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1705403328061789

Reply:
Sorry for not giving more context. PTAL here: https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1706801810707349?thread_ts=1706801099.913429&cid=C02T90PLJ5V

Reply:
gotcha, ive added in the relative imports!